Documentation~/menu.md

```diff
@@ -1,24 +1,172 @@
 # The ProBuilder menu
 
-Use the ProBuilder menu to access most of the ProBuilder editors, actions, tools, and a few special features that are only available through this menu. For example, you can repair and debug ProBuilder, or access the experimental [Boolean operations](boolean.md). 
-
-![The ProBuilder menu](images/pb_menu.png)
-
-The ProBuilder menu items include the following:
-
-* [ProBuilder Window](menu-open.md)
-* [Editors](menu-editors.md)
-* [Dimensions Overlay](menu-dimover.md)
-* [Selection](menu-selection.md)
-* [Interaction](menu-interaction.md)
-* [Object](menu-object.md)
-* [Geometry](menu-geometry.md)
-* [Materials](menu-materials.md)
-* [Vertex Colors](menu-vertexcolors.md)
-* [Experimental](menu-experimental.md) (only available when you enable the [experimental features preference](preferences.md#experimental))
-* [Repair](menu-repair.md)
-* [Export](menu-export.md)
-* [Debug](menu-debug.md)
-* [Actions](menu-actions.md)
+Use the ProBuilder menu to access most of the ProBuilder editors, actions, tools, and features that are only available through this menu. For example, you can repair and debug ProBuilder, or access the experimental [Boolean operations](boolean.md).
 
+## Editors
+
+Create new shapes and open editor windows.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **New Bezier Shape** | Use the **Bezier Shape** tool to define a Bezier curve around which ProBuilder extrudes a Mesh. **Warning:** Use **New Bezier Shape** with caution. This tool is experimental and still under development. If you use this tool, it might reduce ProBuilder's stability. For more information, refer to [Bezier Shape tool](bezier.md). |
+| **New Poly Shape** | Create custom ProBuilder mesh shapes. For more information, refer to [Poly Shape tool](shape-tool.md). | 
+| **New Shape** | Create new ProBuilder mesh shapes such as cylinders, arches, and stairs. For more information, refer to [Shape tool](shape-tool.md). |
+| **Open Lightmap UV Editor** | Change the settings for generating lightmap UVs. For more information, refer to [Lightmap UV Editor](lightmap-uv.md). |
+| **Open Material Editor** | Apply materials to objects or faces. For more information, refer to [Material Editor](material-tools.md). |
+| **Open Smoothing Editor** | Create smooth and rounded or sharp and hard cornered edges. For more information, refer to [Smooth Group Editor](smoothing-groups.md). |
+| **Open UV Editor** | Apply textures to objects or faces. For more information, refer to [UV Editor](uv-editor.md). |
+| **Open Vertex Color Editor** | Apply or paint vertex colors onto meshes. For more information, refer to [Vertex Colors](vertex-colors.md). |
+| **Open Vertex Position Editor** | Enter specific translation coordinates to modify vertex positions. For more information, refer to [Positions Editor](vertex-positions.md). |
+
+## Edit
+
+Edit shapes.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **Edit PolyShape** | |
```

Comment:
@JonMUnity or @modrimkus-unity for some reason these two are never available for me.

Reply:
Seriously, not available for a shape, ever. 
Also I have to wonder why we have this, since the same fields are in the Inspector.

Reply:
Oh I caught something here. 
I never got it for shape because I would right-click > ProBuilder > pick a shape. The Edit Shape option only appears for shapes you created from the main menu, even though it's the same list of shapes.

Reply:
@modrimkus-unity if you create a shape through the Hierarchy's right-click menu, it doesn't ever get the main menu > Tools > ProBuilder > Edit option active.

Reply:
This is definitely a bug! I just reproduced it. @modrimkus-unity, you might want to look into this

Reply:
@suearkinunity, @JonMUnity I see the confusion but this is by design (though not very transparent to the user). The main menu's path to a Cube is : `Tools -> Probuilder -> Editors -> *New Shape* -> Cube`, the hierarchy's `Probuilder -> Cube`, the key difference being the **New Shape** hint. The main menu action creates a procedurally editable PB Shape Mesh where the hierarchy menu a regular PB Mesh so that's why you don't get "Edit Shape" for the later. The hierarchy menu action functions that way to stay in parity with `3D -> Cube` for example, which just instantly spawns a mesh without engaging any creation tool.

In general, whether or not `... -> Edit Shape` is available depends on the presence of `ProBuilderShape` component on the GameObject. Main menu cube vs hierarchy cube components:

<img width="993" alt="PBShapeCube" src="https://github.com/Unity-Technologies/com.unity.probuilder/assets/80063635/cc69ea26-1ebf-4a15-b5d1-fe04ead0631e">

<img width="1012" alt="PBCube" src="https://github.com/Unity-Technologies/com.unity.probuilder/assets/80063635/e3885b05-5f21-4d27-aa3e-28a9173eef28">

Reply:
I see. Thanks for the explanation, @modrimkus-unity .

Reply:
@modrimkus-unity from a user perspective, there are two types of Cubes under ProBuilder, and nothing hinting about that in the Cube name? A difference in the menu layout (ProBuilder > New Shape rather than right-click > ProBuilder) isn't a strong enough hint, since these are two different menus so some layout differences are not meaningful for the user.